.github/workflows/release-python-client.yml

```diff
@@ -4,35 +4,13 @@ on:
   pull_request:
   workflow_call:
     inputs:
-      new-tag:
+      python-version:
```

Comment:
is it confusing to call this `python-version` even though it's coming from the new tag? it's being used as the python version but its source is the new tag, so now I'm rethinking renaming it.

Reply:
could rename it `new-tag-for-python-version`?